core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -276,6 +276,9 @@ class SparkContext(config: SparkConf) extends Logging {
 
   private[spark] def ui: Option[SparkUI] = _ui
 
+  /**
+   * @return `Option` containing URL to web UI.
```

Comment:
This kind of doc doesn't add value, because it doesn't add anything the method signature doesn't already say. I'd remove it or expand it. It's picky, but I think we did assert this before.